### PR TITLE
Translate `library/graphlib.po`

### LINKED ISSUE #373: Translate `library/graphlib.po`

### PR DESCRIPTION
**Pull request by mattwang44**
resolve #373 

image

### CODE REVIEW

File: library/graphlib.po [hunk 1]
```diff
@@ -31,6 +33,8 @@ msgid ""
 "Provides functionality to topologically sort a graph of :term:`hashable` "
 "nodes."
 msgstr ""
+"提供對包含可雜湊 (hashable) 節點之圖 (graph) 進行拓撲排序 (topologically "
```
Comment (mattwang44) [inline library/graphlib.po]:
```suggestion
"提供對包含\\ :term:`可雜湊 (hashable) <hashable>` 節點之圖 (graph) 進行拓撲排序 (topologically "
```

File: library/graphlib.po [hunk 2]
```diff
@@ -52,58 +62,73 @@ msgid ""
...
+"\n"
+"::"
 
 #: ../../library/graphlib.rst:87
 msgid ""
 "Add a new node and its predecessors to the graph. Both the *node* and all "
 "elements in *predecessors* must be :term:`hashable`."
 msgstr ""
+"向圖中新增新節點及其前驅節點。*node* 和 *predecessors* 中的所有元素都必須是可"
+"雜湊的。"
```
Comment (mattwang44) [inline library/graphlib.po]:
```suggestion
"向圖中新增新節點及其前驅節點。*node* 和 *predecessors* 中的所有元素都必須是\\ :term:`可雜湊 <hashable>`\\ "
"的。"
```

File: library/graphlib.po [hunk 3]
```diff
@@ -200,25 +262,31 @@ msgid ""
...
-msgstr ""
+msgstr ":mod:`graphlib` 模組定義了以下例外類別："
 
 #: ../../library/graphlib.rst:203
 msgid ""
 "Subclass of :exc:`ValueError` raised by :meth:`TopologicalSorter.prepare` if "
 "cycles exist in the working graph. If multiple cycles exist, only one "
 "undefined choice among them will be reported and included in the exception."
 msgstr ""
+":exc:`ValueError` 的子類別，如果作用的圖中有循環的存在則由 :meth:"
```
Comment (josix) [inline library/graphlib.po]: **issue:** 
```suggestion
":exc:`ValueError` 的子類別，如果作用的圖中存在循環則由 :meth:"
```

File: library/graphlib.po [hunk 4]
```diff
@@ -52,58 +62,73 @@ msgid ""
...
 msgstr ""
+"如果只需要立即對圖中的節點進行排序且不涉及平行性 (parallelism)，則可以直接使"
+"用便捷方法 :meth:`TopologicalSorter.static_order`："
 
 #: ../../library/graphlib.rst:60
 msgid ""
 "The class is designed to easily support parallel processing of the nodes as "
 "they become ready. For instance::"
 msgstr ""
+"該類別旨在簡單支援節點準備就緒時的平行處理。例如：\n"
```
Comment (josix) [inline library/graphlib.po]: **issue:** 
```suggestion
"該類別設計為在節點準備就緒時，簡單支援節點的平行處理。例如：\n"
```

**Review by josix — CHANGES_REQUESTED** [review]

**Review by josix — APPROVED** [review]
LGTM!

**Pull request merged**